src/parser/shaman/elemental/modules/talents/SurgeOfPower.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+
+import Analyzer from 'parser/core/Analyzer';
+import StatisticBox, { STATISTIC_ORDER } from 'interface/others/StatisticBox';
+import SpellIcon from 'common/SpellIcon';
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import { SELECTED_PLAYER } from 'parser/core/EventFilter';
+import Events from 'parser/core/Events';
+
+const SURGE_OF_POWER = {
+  DURATION: 15000,
+  WINDOW_DURATION: 500,
```

Comment:
Sorry for being so slow with feedback


Duration and window duration are both unused.